Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java

```diff
@@ -152,13 +152,15 @@ public boolean startExport(Process process, URI inZielVerzeichnis, DigitalDocume
                 if (exportDmsTask != null) {
                     exportDmsTask.setException(e);
                 } else {
+                    logger.error(e.getMessage(), e);
                     Helper.setFehlerMeldung("dataCopier.syntaxError", e.getMessage());
                 }
                 return false;
             } catch (RuntimeException e) {
                 if (exportDmsTask != null) {
                     exportDmsTask.setException(e);
                 } else {
+                    logger.error(e.getMessage(), e);
```

Comment:
Please replace this log call and following `Helper.setFehlermeldung` with `Helper.setErrorMessage` which bundles both calls.